Ignore glyph if font does not contain it

smartman opened the pull request:
Some fonts could be incomplete causing Undefined offset: errors to happen when finding elements from glyphs array. For example http://www.fontspace.com/kineticplasma-fonts/hussar-bold-web-edition causes 'Undefined offset: 132'

kartsims commented:
nice job man, thanks !

could you please fix the indentation so that I can merge the PR ?

smartman committed:
Ignore glyph if font does not contain it

Some fonts could be incomplete causing Undefined offset: errors to happen when finding elements from glyphs array. For example http://www.fontspace.com/kineticplasma-fonts/hussar-bold-web-edition causes 'Undefined offset: 132'

smartman commented:
All fixed man, code style is in alignment with rest of your code now.

kartsims commented:
Awesome.

Thank you !
